server/build/webpack.js

```diff
@@ -299,10 +299,11 @@ export default async function createCompiler (dir, { dev = false, quiet = false,
   }
 
   if (!dev) {
+    // We do this to use the minified version of React in production.
+    // This will significant file size redution when turned off uglifyjs.
     webpackConfig.resolve.alias = {
       'react': require.resolve('react/dist/react.min.js'),
-      'react-dom': require.resolve('react-dom/dist/react-dom.min.js'),
-      'react-dom/server': require.resolve('react-dom/dist/react-dom-server.min.js')
```

Comment:
Got it. Sorry, didn't realize Next.js stopping running all tests before commits.

Seems like React team is aware of the issue; it's in multiple places but some of the tickets have a `Won't fix` label on them. I checked `15.4.0` to `15.5.4` and the error exists in all of them.

Would you be open to compiling a server bundle (as part of `next build`)? This way you can apply all the proper optimizations ahead of time... and _might_ even be able to remove Babel regenerator/runtime which would be huge!